entity-management-definitions/src/main/java/eu/europeana/entitymanagement/normalization/EntityFieldsValidator.java

```diff
@@ -0,0 +1,299 @@
+		}
+        else if (field.getType().isAssignableFrom(String.class)){
+			returnValue = validateEmailOrDateOrURIStringField(context, field, fieldValue, returnValue);
+        }
+        return returnValue;
+    }
+    
+    boolean validateEmailOrDateOrURIArrayField(ConstraintValidatorContext context, Field field, Object fieldValue, boolean returnValue) {
+		Object[] fieldValueArray = (Object[])fieldValue;    			
+		if (fieldValueArray.length==0) return returnValue;			
+		for (Object fieldValueElem : fieldValueArray) {
+			if (EntityFieldsTypes.valueOf(field.getName()).getFieldType().equals("Email")) {
+				returnValue = checkEmailFormat(context, field, fieldValueElem, returnValue);
+			}
+			else if (EntityFieldsTypes.valueOf(field.getName()).getFieldType().equals("Date")) {
+				returnValue = checkDateFormatISO8601(context, field, fieldValueElem, returnValue);
+			}
+			else if (EntityFieldsTypes.valueOf(field.getName()).getFieldType().equals("URI")) {
+				returnValue = checkURIFormat(context, field, fieldValueElem, returnValue, true);
+			}
+		}
+        return returnValue;
+    }  
+    
+    boolean validateEmailOrDateOrURIListField(ConstraintValidatorContext context, Field field, Object fieldValue, boolean returnValue) {
+ 		@SuppressWarnings("unchecked")
+		List<Object> fieldValueList = (List<Object>)fieldValue;
+		if (fieldValueList.size()==0) return returnValue;
+		for (Object fieldValueElem : fieldValueList) {
+			if (EntityFieldsTypes.valueOf(field.getName()).getFieldType().equals("Email")) {
+				returnValue = checkEmailFormat(context, field, fieldValueElem, returnValue);
+			}
+			else if (EntityFieldsTypes.valueOf(field.getName()).getFieldType().equals("Date")) {
+				returnValue = checkDateFormatISO8601(context, field, fieldValueElem, returnValue);
+			}
+			else if (EntityFieldsTypes.valueOf(field.getName()).getFieldType().equals("URI")) {
+				returnValue = checkURIFormat(context, field, fieldValueElem, returnValue, true);					
+			}
+			
+		}		
+        return returnValue;
+    }
+    
+    boolean validateEmailOrDateOrURIStringField(ConstraintValidatorContext context, Field field, Object fieldValue, boolean returnValue) {
+    	if (EntityFieldsTypes.valueOf(field.getName()).getFieldType().equals("Email")) {
+    		returnValue = checkEmailFormat(context, field, fieldValue, returnValue);
+    	}
+    	else if (EntityFieldsTypes.valueOf(field.getName()).getFieldType().equals("Date")) {
+    		returnValue = checkDateFormatISO8601(context, field, fieldValue, returnValue);
+    	}
+    	else if (EntityFieldsTypes.valueOf(field.getName()).getFieldType().equals("URI")) {
+			returnValue = checkURIFormat(context, field, fieldValue, returnValue, true);				
+		}
+        return returnValue;
+    }
+    
+    boolean checkEmailFormat(ConstraintValidatorContext context, Field field, Object fieldValue, boolean returnValue) {
+        String emailRegex = "^[a-zA-Z0-9_+&*-]+(?:\\."+
```

Comment:
please use the EmailValidator fro apache commons: https://commons.apache.org/proper/commons-validator/apidocs/org/apache/commons/validator/routines/EmailValidator.html

Reply:
Done.